WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1803,3 +1805,12 @@ FileTypeCal         = @FileIconCommonStyle@ background-color:#94385d;
 FileTypeVCard       = @FileIconCommonStyle@ background-color:#666666;
 FileTypeMedia       = @FileIconCommonStyle@ background-color:#9e61ca;
 FileTypeMail        = @FileIconCommonStyle@ background-color:@PrimaryBrandingColor@
+
+#######################################################################
+#   Create Contact-Group Dialog
+#######################################################################
+
+CreateContactGroupDialog        =
```

Comment:
No problem with having an empty placeholder. That enables developer in future to have styling really 👍  . 
Suggestion was on a thought that should we have all dialogs styling at once place ? Maybe in future commit .